cluster/aws/util.sh

```diff
@@ -360,6 +360,10 @@ function detect-trusty-image () {
         AWS_IMAGE=ami-93876e93
         ;;
 
+      ap-northeast-2)
+        AWS_IMAGE=ami-62ac620c
```

Comment:
For the record, I verified this ID against https://cloud-images.ubuntu.com/locator/